ftl/python/testdata/functionality_test/start.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 set -xe
-python /srv/app.py &
-wget --tries=20 localhost:8080
+if [ "$(cat /workspace/ftl/python/testdata/additional_directory/child/b.txt)" = "barfoo" ]; then
```

Comment:
you shouldn't have to mess with the functionality_test, that is supposed to only test that an app runs.  For something like testing whether a folder exists, you should add this as a part of the integration tests:
https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/ftl/integration_tests/ftl_python_integration_tests_yaml.py#L9

then create a folder for the test w/ a structure_test.yaml (see: https://github.com/GoogleCloudPlatform/runtimes-common/tree/master/ftl/python/testdata/packages_test)